### ISSUE TITLE
CountDays=true - control is invoked each time on selected day

### ISSUE THREAD

**Issue by tlucz** [microsoft/rate-my-app#32]
Hi,
I use your control in my WP8 app. I set CountDays=true so only one app launch is count per day. But there is a problem on days which are selected to display 'FeedbackOverlay' control. I set FirstCount on 3, so on third day on every app launch control is invoked. In my opinion it is wrong behaviour and control should be visible only once a day.

br,
Tadeusz

**Comment by ltomuta** [comment]
True, the behavior should be as you describe it. It sounds like we have a bug.

Sent from my Nokia Lumia 920 powered by Windows Phone 8
